pvlib/irradiance.py

```diff
@@ -1776,7 +1776,7 @@ def dirint(ghi, solar_zenith, times, pressure=101325., use_delta_kt_prime=True,
         GHI points is 1.5 hours or greater. If use_delta_kt_prime=True,
         input data must be Series.
 
-    temp_dew : None, float, or array-like, default None
+    temp_dew : None, float, or array-like, optional
         Surface dew point temperatures, in degrees C. Values of temp_dew
```

Comment:
Here and throughout, ``temp_dew`` in the docstrings. This is an optional nit, decline if you feel it's out of scope for the PR.

Reply:
Done!